app/components/page-components/Chatbot/ChatbotCommandVariables.vue.ts

```diff
@@ -1,10 +1,50 @@
 import ChatbotBase from 'components/page-components/Chatbot/ChatbotBase.vue';
 import { Component } from 'vue-property-decorator';
+import { Accordion, Badge } from 'streamlabs-beaker';
+import mapValues from 'lodash/mapValues';
+import pickBy from 'lodash/pickBy';
+@Component({
+  components: { Accordion, Badge },
+})
+export default class ChatbotCommandVariables extends ChatbotBase {
+  searchQuery = '';
 
-@Component({})
-export default class ChatbotDefaultCommands extends ChatbotBase {
-  get variables() {
-    return this.chatbotApiService.Commands.state.commandVariablesResponse;
+  get filteredVariables() {
+    const grouped = {};
+    const arr = this.chatbotApiService.Commands.state.commandVariablesResponse as any[];
+
+    for (const variable of arr) {
+      for (const tag of variable.tags) {
+        if (!grouped[tag]) {
+          grouped[tag] = [];
+          grouped[tag].push(variable);
+        } else {
+          grouped[tag].push(variable);
+        }
+      }
+    }
+
+    const filteredVariables = mapValues(grouped, (section, slug) => {
+      return pickBy(
+        mapValues(section, (variable: any) => {
+          const found =
+            slug.toLowerCase().indexOf(this.searchQuery.toLowerCase()) > -1 ||
+            variable.variable.toLowerCase().indexOf(this.searchQuery.toLowerCase()) > -1 ||
+            variable.example.toLowerCase().indexOf(this.searchQuery.toLowerCase()) > -1 ||
+            variable.result.toLowerCase().indexOf(this.searchQuery.toLowerCase()) > -1;
```

Comment:
```
const found = [slug, variable.variable, variable.example, variable.result]
 .map(str => str.toLowerCase())
 .find(str => str.includes(this.searchQuery.toLowerCase()))
```

Reply:
Done, it's in the latest commit. Ty for letting me know of a cleaner way to do this :)